tests/console-backend-service/internal/domain/k8s/namespace_test.go

```diff
@@ -25,17 +25,26 @@ func TestNamespace(t *testing.T) {
 
 	suite := givenNewTestNamespaceSuite(t)
 
+	t.Log("Creating namespace...")
 	createRsp, err := suite.whenNamespaceIsCreated()
 	suite.thenThereIsNoError(t, err)
 	suite.thenThereIsNoGqlError(t, createRsp.GqlErrors)
 	suite.thenCreateNamespaceResponseIsAsExpected(t, createRsp)
 	suite.thenNamespaceExistsInK8s(t)
 
+	t.Log("Quering for namespace...")
 	queryRsp, err := suite.whenNamespaceIsQueried()
 	suite.thenThereIsNoError(t, err)
 	suite.thenThereIsNoGqlError(t, queryRsp.GqlErrors)
 	suite.thenNamespaceResponseIsAsExpected(t, queryRsp)
 
+	t.Log("Updating namespace...")
+	updateResp, err := suite.whenNamespaceIsUpdated()
+	suite.thenThereIsNoError(t, err)
+	suite.thenThereIsNoGqlError(t, updateResp.GqlErrors)
+	suite.thenUpdateNamespaceResponseIsAsExpected(t, updateResp)
+
```

Comment:
Verify by k8s client if labels have been set.

Reply:
We're checking if the new object has the `updatedLabels` field, so do we need that additional step?

Reply:
Yes. `updateLabels` is returned from CBS, but we need to check in k8s if the labels have been applied.